ksqldb-execution/src/main/java/io/confluent/ksql/execution/expression/tree/LambdaFunctionExpression.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.execution.expression.tree;
+
+import static java.util.Objects.requireNonNull;
+
+import com.google.common.collect.ImmutableList;
+import io.confluent.ksql.parser.NodeLocation;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+
+
+public class LambdaFunctionExpression extends Expression {
```

Comment:
Not a comment just a question regarding the class names: I saw the closest extension of `Expression` to `LambdaFunctionExpression` is called `FunctionCall`, which seems inconsistent in their naming. Is there any rationale behind this? Or should we either have:

* RegularFunctionCall, LambdaFunctionCall

Or:

* RegularFunctionExpression, LambdaFunctionException

Reply:
I changed LambdaFunctionExpression to LambdaFunctionCall